### PR TITLE
feat: initialize MUI theme

### LINKED ISSUE #4: Initialize MUI theme
**As a** developer, 
**I want** to initialize MUI theme, 
**so that I can** use theme settings for react UI (presentation) components development.

### Solution notes
- investigate new MUI version features to be applicable in project

### Acceptance Criteria

- MUI theme is configured to use fields:
  - [.palette](https://mui.com/material-ui/customization/palette/)
  -  [.typography](https://mui.com/material-ui/customization/typography/)
  -  [.spacing](https://mui.com/material-ui/customization/spacing/)
  -  [.breakpoints](https://mui.com/material-ui/customization/breakpoints/)

### Resources:

* Figma: https://www.figma.com/file/jRZ9Rde5hBU2Ujitx1lQjX/Books-club?node-id=0%3A1&t=kfz2vC4CT4OoDe4k-0
* MUI - https://mui.com

### Notes

breakpoints: 1280 - 768 - 320

### PR DESCRIPTION
**Pull request by OlyaPolya**
### Task #4 

* [x] `npm run lint` check passed without any failures or warnings
* [ ] functionality is covered by new tests
* [x] functionality works in production build
* [ ] existing functionality remains unchanged
* [ ] updated documentation (if required)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,39 +1,60 @@
...
+  },
+  "lint-staged": {
+    "*.{js,tsx,ts}": "eslint --fix"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@fontsource/montserrat": "^4.5.14",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/material": "^5.11.8",
```
Comment (AnhelinaVr) [inline package.json]: need to ensure if that's prod dependencies

File: package.json [hunk 2]
```diff
@@ -1,39 +1,60 @@
...
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "husky": "^8.0.0",
+    "lint-staged": "^13.1.2",
+    "prettier": "^2.8.3"
+  },
+  "browserslist": {
```
Comment (AnhelinaVr) [inline package.json]: please move this field above the dependencies

**Review by Konstantsin-Shumak — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (OlyaPolya) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (OlyaPolya) [reply package.json]: Done

**Review by AnhelinaVr — APPROVED** [review]
lgtm

**Pull request merged**